### DISCUSSION TITLE
atomic_dipole file

### DISCUSSION THREAD

**Discussion by hanao2** [deepmodeling/deepmd-kit#2557] [General]
Hello,

I have a question regarding the long-range DP model. There is the "atomic_dipole.npy" file that we need to supply. Does this file include 1)molecular WCs or 2)molecular dipoles ((q_N)(r_N)+(q_WC)(r_WC))?

Thanks!

**Comment by wanghan-iapcm** [comment]
It supplies the displacement of the wannier centroid respect to the associated atom.
